extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -575,9 +585,19 @@ export class AmpA4A extends AMP.BaseElement {
           const extensions = extensionsFor(this.win);
           creativeMetaDataDef.customElementExtensions.forEach(
             extensionId => extensions.loadExtension(extensionId));
+          // Make sure we don't collapse this slot.
+          creativeMetaDataDef.collapse = false;
           return creativeMetaDataDef;
         })
         .catch(error => {
+          if (error == NO_CONTENT_RESPONSE) {
+            return {
+              minifiedCreative: null,
```

Comment:
I'm not sure that this obeys the type def for `CreativeMetaDataDef` -- I don't _think_ this field is nullable, given that spec.  (Run the type checker to be sure, though.)  If necessary, use `''` here?

Also, check types in following lines.